### PR TITLE
[17.0][IMP] fs_product_multi_image: Replace tree view with kanban view

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**

### CODE REVIEW

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM 👍🏿

File: fs_product_multi_image/views/product_template.xml [hunk 1]
```diff
@@ -28,17 +28,25 @@
                                 nolabel="1"
                                 colspan="2"
                             >
-                                <tree>
-                                    <field name="sequence" widget="handle" />
-                                    <field name="name" />
-                                </tree>
+                            <kanban>
+                                <field name="name" />
```
Comment (lmignon) [inline fs_product_multi_image/views/product_template.xml:32]: How do you manage sequence now? Do we really need to always display the images?  If you take a look at the code in version 16, a new widget has been developped with a lot of usefull functionnality. We should forward port https://github.com/OCA/storage/pull/332 instead.

**Review by lmignon — CHANGES_REQUESTED** [review]
Thak you for the contribution. IMO we should forward port https://github.com/OCA/storage/pull/332

File: fs_product_multi_image/views/product_template.xml (hunk 1 above)
Comment (JasminSForgeFlow) [reply fs_product_multi_image/views/product_template.xml:32]: You are right

**Pull request closed without merging**